core/services/ocr2/delegate.go

```diff
@@ -1568,17 +1570,12 @@ func (d *Delegate) newServicesOCR2Functions(
 	if err != nil {
 		return nil, fmt.Errorf("functions services: failed to get chain %s: %w", rid.ChainID, err)
 	}
+	rargs := d.relayConfigsWithDefaults(jb)
 	createPluginProvider := func(pluginType functionsRelay.FunctionsPluginType, relayerName string) (evmrelaytypes.FunctionsProvider, error) {
 		return evmrelay.NewFunctionsProvider(
 			ctx,
 			chain,
-			types.RelayArgs{
-				ExternalJobID: jb.ExternalJobID,
-				JobID:         jb.ID,
-				ContractID:    spec.ContractID,
-				RelayConfig:   spec.RelayConfig.Bytes(),
-				New:           d.isNewlyCreatedJob,
-			},
+			rargs,
```

Comment:
Since the helper func is only modifying the `RelayerConfig` and no other fields on the `RelayArgs`, what about reducing it's scope to something like this?
```suggestion
			types.RelayArgs{
				ExternalJobID: jb.ExternalJobID,
				JobID:         jb.ID,
				ContractID:    spec.ContractID,
				RelayConfig:   d.relayConfigWithDefaults(spec.RelayConfig),
				New:           d.isNewlyCreatedJob,
			}
```